app/models/tasks/pre_docket/vha_document_search_task.rb

```diff
@@ -25,7 +25,7 @@ def available_actions(user)
   ].freeze
 
   VHA_CAREGIVER_SUPPORT_TASK_ACTIONS = [
-    # Documents ready for Board Intake review,
+    Constants.TASK_ACTIONS.VHA_CAREGIVER_SUPPORT_DOCUMENTS_READY_FOR_BOARD_INTAKE_REVIEW.to_h
     # Return to Board Intake
```

Comment:
```suggestion
    # Return to Pizza Hut
```
Hey I think this is better